### PR TITLE
[ci] Add test capture

### PR DESCRIPTION
**Pull request by vrozic**
Test PR adds the first FPGA test to the CI.
The test runs `aes-fvsr-key-batch` command, collects 1000 measurements on CW310 and publishes the collected traces.

### CODE REVIEW

**Comment by pamaury** [comment]
There are lots of things missing due to how azure works, here is what it should look like probably:
```yaml
- job: sca_capture
  displayName: "Capture SCA traces"
  timeoutInMinutes: 30
  dependsOn: checkout
  pool: FPGA SCA
  steps:
  - checkout: self
  - bash: |
      python3 -m pip install --user -r python-requirements.txt
    displayName: "Install python dependencies"
  - bash: |
      apt install ... # if any
    displayName: "Install system dependencies" 
  - bash: |
      pushd ci
      ./ci_capture_aes_fvsr.sh
      popd
    displayName: "Capture traces" 
  # here migth want to add more steps to publish the traces, e.g.
  - publish: path/to/file
    artifact: traces
    displayName: "Upload traces"
```

**Comment by pamaury** [comment]
also it might need to setup git-lfs if you use it for the tests

**Comment by jwnrt** [comment]
> ```yaml
>   - bash: |
>       apt install ... # if any
>     displayName: "Install system dependencies" 
> ```

Based on the getting-started guide, I think this should be `sudo apt install git-lfs`.

File: ci/azure-pipelines.yml [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
+  - bash: |
+      sudo apt install git-lfs
+    displayName: "Install system dependencies"
+  - bash: |
+      pushd ci
+      ./ci_capture_aes_fvsr.sh
+      popd
+    displayName: "Capture traces"
+  # here migth want to add more steps to publish the traces, e.g.
+  - publish: path/to/file
```
Comment (jwnrt) [inline ci/azure-pipelines.yml:25]:
```suggestion
  - publish: ./ci/ci_projects/
```

File: ci/azure-pipelines.yml [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
 
 jobs:
-- job: hello_world
-  displayName: Hello world
-  pool:
-    vmImage: ubuntu-latest
+- job: sca_capture
+  displayName: "Capture SCA traces"
+  timeoutInMinutes: 30
+  dependsOn: checkout
```
Comment (jwnrt) [inline ci/azure-pipelines.yml:10]:
```suggestion
```

Sorry, we need to remove this line

**Comment by jwnrt** [comment]
Is the directory definitely called `opentitan_simple_aes_data` when the project name is `opentitan_simple_aes` (without the `_data`)?

**Review by jwnrt — APPROVED** [review]

**Review by andreaskurth — APPROVED** [review]
LGTM, thanks @vrozic for implementing this and @pamaury and @jwnrt for your feedback :+1:

**Comment by jwnrt** [comment]
Thanks for iterating on this with us @vrozic.

Artifacts are available in the Azure interface for each job, e.g. [this run](https://dev.azure.com/lowrisc/ot-sca/_build/results?buildId=129706&view=logs&j=6d169879-57a5-53d8-358a-1e9c96dbeb21]) as a link in the logs:

> 1 artifact produced

which links to [this artifact](https://dev.azure.com/lowrisc/ot-sca/_build/results?buildId=129706&view=artifacts&pathAsName=false&type=publishedArtifacts). Not the easiest to discover, I know.

**Pull request merged**